static/app/views/insights/browser/webVitals/components/webVitalMetersWithIssues.tsx

```diff
@@ -0,0 +1,337 @@
+import React, {Fragment, useState} from 'react';
+import {useTheme} from '@emotion/react';
+import styled from '@emotion/styled';
+import * as qs from 'query-string';
+
+import {LinkButton} from 'sentry/components/core/button/linkButton';
+import {Tooltip} from 'sentry/components/core/tooltip';
+import InteractionStateLayer from 'sentry/components/interactionStateLayer';
+import ExternalLink from 'sentry/components/links/externalLink';
+import QuestionTooltip from 'sentry/components/questionTooltip';
+import {IconIssues} from 'sentry/icons';
+import {t, tct} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {Organization} from 'sentry/types/organization';
+import useOrganization from 'sentry/utils/useOrganization';
+import {PerformanceBadge} from 'sentry/views/insights/browser/webVitals/components/performanceBadge';
+import {VITAL_DESCRIPTIONS} from 'sentry/views/insights/browser/webVitals/components/webVitalDescription';
+import {WEB_VITALS_METERS_CONFIG} from 'sentry/views/insights/browser/webVitals/components/webVitalMeters';
+import {
+  getIssueQueryFilter,
+  useWebVitalsIssuesQuery,
+} from 'sentry/views/insights/browser/webVitals/queries/useWebVitalsIssuesQuery';
+import {MODULE_DOC_LINK} from 'sentry/views/insights/browser/webVitals/settings';
+import {
+  type ProjectScore,
+  WEB_VITAL_PERFORMANCE_ISSUES,
+  type WebVitals,
+} from 'sentry/views/insights/browser/webVitals/types';
+
+export type ProjectData = {
+  'p75(measurements.cls)': number;
+  'p75(measurements.fcp)': number;
+  'p75(measurements.inp)': number;
+  'p75(measurements.lcp)': number;
+  'p75(measurements.ttfb)': number;
+};
+
+type Props = {
+  onClick?: (webVital: WebVitals) => void;
+  projectData?: ProjectData[];
+  projectScore?: ProjectScore;
+  showTooltip?: boolean;
+  transaction?: string;
+};
+
+export default function WebVitalMetersWithIssues({
+  onClick,
+  projectData,
+  projectScore,
+  showTooltip = true,
+}: Props) {
+  const theme = useTheme();
+  if (!projectScore) {
+    return null;
+  }
+
+  const webVitalsConfig = WEB_VITALS_METERS_CONFIG;
```

Comment:
I think you can omit this variable assignment, it's not really clarifying anything